client/stylesheet.css

```diff
@@ -0,0 +1,119 @@
+@import url('https://fonts.googleapis.com/css2?family=Mulish:ital,wght@1,200&family=Roboto&family=Varta&display=swap');
+
+body {
+    background-color: white;
+    font-family: 'Roboto', sans-serif;
+  }
+
+
+.form{
+    text-align: center;
+}
+
+.footer-left{
+    background-color: grey;
+}
+
+header{
+    padding-top: 10px;
+    text-align: center; 
+    font-size: 100%;
+}
+
+h1{
+    padding-top: 100px;
+    text-align: center;
+}
+
+.bigtext{
+    font-size: 35px;
+}
+
+.form {
+    text-align: center;
+  }
+
+#form-search { 
+    width: 450px;
+    line-height: 0px;
+    padding: 5px 10px;
+  }
+
+.buttons {
+    text-align: center;
+    padding-top: 10px;
+    margin-bottom: 300px;
+  }
+
+
+#nav_bar {
+    float: right;
+  }
+  
```

Comment:
float still works well for positioning elements but I would encourage to use flex box as a modern css styling. Specially when it comes to nav bar I really think it make the things easier.